---
title: Known Pathogenic STRs in gnomAD v3.1
date: 2021-10-01
order: 1
categories:
  - Announcements
authors:
  - Ben Weisburd
  - Nick Watts
---


### Overview

We have generated Short Tandem Repeat (STR) calls for 60 known pathogenic loci by running ExpansionHunter on 20,344 whole genome samples from gnomAD v3.1. This data can be explored through the 60 STR variant pages listed in the Table of STR Loci \[link].

The release includes:

* Distributions of STR genotypes at these loci in the general population. These distributions can be subsetted by 10 gnomAD subpopulations, sex, and/or the repeat unit (for loci where more than one repeat unit exists in the population).
* Read visualizations for all samples above the pathogenic threshold of each locus \[TODO is this accurate?].
* Reference information for each known pathogenic STR locus - including its pathogenic threshold, disease association, and inheritance mode.
* Downloadable variant catalog \[link] in .json format which can be used to run ExpansionHunter using the same parameters as the ones used to generate this callset.

<!-- end_excerpt -->

### Background

About STRs

### Details

Details

### Navigating the STR Page

Walk-through

### REViewer Visualizations

Read Visualizations

### In Summary

Summaray

### Acknowledgments

We thank Stephanie DiTroia, Grace VanNoy, and Nehir Kurtas for reviewing the list of known pathogenic loci and evaluating disease associations; Egor Dolzhenko for consultations on various topics related to ExpansionHunter, REViewer, and STRs; Depienne et al. and Halman et al. authors for the very helpful resources; Yossi Farjoun for mentorship during the initial stages of the project; Jackie Goldstein, Daniel King, Cotton Seed and the rest of the hail team for building Hail Batch and other tools that made it much easier to generate this callset.

